### ISSUE TITLE
I was unable to build, so I update the dependencies and everything worked.

### ISSUE THREAD

**Issue by longtimedeveloper** [Chinmay-KB/route_guard_example#1]
Outstanding example on `auto_route` thank you!

After downloading this repo, I keep getting strange errors on the build.

I tried running this command in the terminal but still got errors:
  `flutter packages pub run build_runner build    --delete-conflicting-outputs`

So, I updated the dependencies to:
  `auto_route: ^3.2.0`
  `auto_route_generator: ^3.2.0`
  `build_runner: ^2.1.7`

Then I ran this command in the terminal:
  `flutter packages pub run build_runner build    --delete-conflicting-outputs`

App ran perfectly!

Thank you again for your work in putting this together!  Very easy to understand the concepts and your article is splendid!

Karl

**Comment by Chinmay-KB** [comment]
I think the issue you were facing was because of [this issue](https://github.com/Milad-Akarie/auto_route_library/issues/878) in auto route I had filed earlier and you may be running flutter 2.8.  Thanks for pointing it out, I'll update the example or if you can send a pull request, that'd be awesome 😁

**Comment by longtimedeveloper** [comment]
@Chinmay-KB yes, `Flutter (Channel stable, 2.8.1, on Microsoft Windows [Version 10.0.19043.1415], locale en-US)`

You can close this if you want since you're adding notes for developers.

Even with these minor Flutter version issues with packages, it is still MUCH better than Window DLL hell, NuGet package hell, or Angular npm package problems.  One of the biggest reasons I've switched to Flutter!

Best to you, Karl

**Comment by vinitmp15** [comment]
Hey @Chinmay-KB. I think you should mention this issue in your readme.md. I would be quite helpful.

Good work BTW
